content/en/database_monitoring/setup_sql_server/azure.md

```diff
@@ -71,6 +73,21 @@ GRANT VIEW SERVER STATE to datadog;
 GRANT VIEW ANY DEFINITION to datadog;
 ```
 
+#### For SQL Server 2012
+
+```SQL
+CREATE LOGIN datadog WITH PASSWORD = '<PASSWORD>';
+CREATE USER datadog FOR LOGIN datadog;
+GRANT VIEW SERVER STATE to datadog;
+GRANT VIEW ANY DEFINITION to datadog;
+```
+
+Create the `datadog` user in each additional application database:
+```SQL
+USE [database_name];
+CREATE USER datadog FOR LOGIN datadog;
+```
```

Comment:
I ended up doing this because I couldn't see to get a nested tab to work right 
![Screen Shot 2022-05-27 at 11 36 39 AM](https://user-images.githubusercontent.com/14317240/170770872-b22e9749-9e00-4b34-9bf2-6afbcaa64fab.png)